conf.yaml

```diff
@@ -1600,6 +1620,26 @@ contents:
     -
         title:      Legacy Documentation
         sections:
+          -
+            title:      Infrastructure Monitoring Guide for 6.5-7.4
+            prefix:     en/infrastructure/guide
+            current:    7.4
+            branches:   [ 7.x, 7.5, 7.4, 7.3, 7.2, 7.1, 7.0, 6.8, 6.7, 6.6, 6.5 ]
```

Comment:
Do we still want to build `7.x` and `7.5` here? `7.4` will be the last current so might make sense not to build those other branches.

If we do want to build them, you should update the title in L1624

Reply:
Continuing to build 7.x and 7.5 would be really confusing, I think. 

When we've refactored docs in the past, folks have had a hard time figuring out how to get to the latest docs. I think you should customize the banner message to point to the new docs. (Not everyone will read it, but it will help some folks.)

Reply:
I've had to leave 7.5 and 7.x for now, because otherwise the 7.5 and 7.x builds break. I can't remove these until I've merged the related changes in Kibana, beats and elsewhere in stack-docs into the 7.5 and 7.x branches to refer to the new docs instead of this one. I'll move the 7.5 and 7.x builds to the new docs as soon as there is nothing in the 75 and 7.x branches that refers to the old docs.

Reply:
@debadair Thanks for the clarification (via Slack). I've created https://github.com/elastic/stack-docs/issues/645 to address this separately so that I can progress this mega-merge without any more hold-ups.